pkg/printers/internalversion/printers.go

```diff
@@ -3267,19 +3268,33 @@ func printDeviceTaintRuleList(list *resource.DeviceTaintRuleList, options printe
 	return rows, nil
 }
 
-func printStorageVersionMigration(obj *svmv1alpha1.StorageVersionMigration, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+func printStorageVersionMigration(obj *storagemigration.StorageVersionMigration, options printers.GenerateOptions) ([]metav1.TableRow, error) {
 	row := metav1.TableRow{
 		Object: runtime.RawExtension{Object: obj},
 	}
+	migrationGR := obj.Spec.Resource.String()
+	row.Cells = append(row.Cells, obj.Name, migrationGR)
 
-	migrationGVR := obj.Spec.Resource.Resource + "." + obj.Spec.Resource.Version + "." + obj.Spec.Resource.Group
-	row.Cells = append(row.Cells, obj.Name, migrationGVR)
-	//ToDo: add migration condition 'status' and 'type' (migration successful | failed)
+	condStatus := ""
```

Comment:
initialize to something so if we don't have any of these conditions, we don't output an empty value
```suggestion
	condStatus := "Unknown"
```

Reply:
Thanks, wasn't sure whether to leave this empty or put unknown.